pmm-app/src/pmm-settings/components/PlatformLogin/SignUp.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { FC } from 'react';
+import { Field, Form, FormRenderProps } from 'react-final-form';
+import { LinkButton, useTheme } from '@grafana/ui';
+import validators from 'shared/components/helpers/validators';
+import { ButtonWithSpinner } from 'shared/components/Form';
+import { showErrorNotification, showSuccessNotification } from 'shared/components/helpers';
+import { Credentials, LoginFormProps } from './types';
+import { Messages } from './PlatformLogin.messages';
+import { CheckboxFieldAdapter, InputFieldAdapter } from './FieldAdapters/FieldAdapters';
+import { getStyles } from './PlatformLogin.styles';
+import { PlatformLoginService } from './PlatformLogin.service';
+import { PRIVACY_POLICY_URL, TERMS_OF_SERVICE_URL } from './PlatformLogin.constants';
+
+export const SignUp = ({ changeMode, getSettings }: LoginFormProps) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  const handleSignUpFormSubmit = async (credentials: Credentials) => {
+    try {
+      await PlatformLoginService.signUp(credentials);
+
+      getSettings();
```

Comment:
See previous comment about `getSettings`

Reply:
The same as for previous, I'm not sure if we have more elegant solution here. @nicolalamacchia maybe you have any ideas? if yes, please feel free to add it

Reply:
I was thinking about moving out of these components the logic and have `signIn` and `singUp` functions in the `PlatformLogin` component, having those components as "dumb" components, in order to avoid that prop drilling and maybe also share more code